app.js

```diff
@@ -503,11 +503,11 @@ function setupModelRoutes(mongoConnection) {
 
 		res.status(err.status || 500);
 
-		if (req.xhr)
-			return res.json({ message: err.message });
+		if (req.xhr || (req.query.ajax === '1'))
```

Comment:
we should just add the `X-Requested-With` header, which is present anyway with jQuery by default I think